src/config/aragon.ts

```diff
@@ -1,13 +1,16 @@
 import { ConfigExtender } from '@nomiclabs/buidler/types'
 import { AragonConfig } from '~/src/types'
-import { defaultIpfsGateway } from '~/src/params'
+import { defaultIpfsGateway, defaultLocalAragonBases } from '~/src/params'
 
 export const defaultAragonConfig: AragonConfig = {
   appServePort: 8001,
   clientServePort: 3000,
   appBuildOutputPath: 'dist/',
   ignoreFilesPath: '.',
-  ipfsGateway: defaultIpfsGateway
+  ipfsGateway: defaultIpfsGateway,
+  ensAddress: defaultLocalAragonBases.ensAddress,
+  apmAddress: defaultLocalAragonBases.apmAddress,
+  daoFactoryAddress: defaultLocalAragonBases.daoFactoryAddress
```

Comment:
Perhaps we can namespace under an `deployedAddresses` object?

E.g.

```js
config: {
  ...
  deployedAddresses: {
    ens: '',
    daoFactory: '',
  }
}
```

Reply:
I was tempted to do something similar, thanks for the feedback